docs/compilers/CSharp/Compiler Breaking Changes - DotNet 9.md

```diff
@@ -44,3 +44,47 @@ unsafe class C // unsafe context
 ```
 
 You can work around the break simply by adding the `unsafe` modifier to the local function.
+
+## Collection expression breaking changes in C# 13 and newer
```

Comment:
>breaking changes in [](http://example.com/codeflow?start=25&length=19)

Consider clarifying these are specific to overload resolution: "breaking changes with overload resolution in ..."